pkg/qliksense/context_configs.go

```diff
@@ -225,8 +225,40 @@ func (q *Qliksense) SetOtherConfigs(args []string) error {
 			}
 			qliksenseCR.Spec.RotateKeys = rotateKeys
 			api.LogDebugMessage("Current rotateKeys after modification: %s ", qliksenseCR.Spec.RotateKeys)
+		case "gitops.enabled":
+			if qliksenseCR.Spec.GitOps == nil {
+				qliksenseCR.Spec.GitOps = &config.GitOps{}
+			}
+			if strings.EqualFold(argsString[1], "false") {
+				qliksenseCR.Spec.GitOps.Enabled = false
```

Comment:
change its type to string and take value "yes"/"no"
also modify this https://github.com/qlik-oss/k-apis/blob/a9288454f9365a12523819765a1c4e5ac043a146/pkg/config/type.go#L79